webapp-resources/messages_fi.properties

```diff
@@ -2963,6 +2963,8 @@ systemSettings.environmentSettings=Scada-LTS environment settings
 systemSettings.smsDomain=SMS Domain
 systemSettings.setInNewUI=Set in the new UI
 dox.newUISettings=Settings to set in the new UI
+logo.for=varten
+logo.powered=Voimanlähteenä
```

Comment:
I don't think the translation is correct - "Voimanlähteenä" translates to "As a source of power".

"Toimittamat" (Provided by) seems to be a better alternative.